common/src/main/java/gov/cms/ab2d/common/model/Contract.java

```diff
@@ -44,4 +48,35 @@
 
     @OneToMany(mappedBy = "contract")
     private Set<Coverage> coverages = new HashSet<>();
+
+    public boolean hasAttestation() {
+        return attestedOn != null;
+    }
+
+    public void clearAttestation() {
+        attestedOn = null;
+    }
+
+    public void markAttested(OffsetDateTime attestedOnDate) {
```

Comment:
You can deleted this, there already is a setter with lombok

Reply:
Done - it was an artifact of a fancier implementation (with another persisted field).